packages/swarm-docs/src/components/radioExample.js

```diff
@@ -1,35 +1,64 @@
-import React from 'react'
-import { Radio } from '@meetup/swarm-components'
+import React from 'react';
+import { Radio } from '@meetup/swarm-components';
 
 const Example = () => {
-  const [checked, setChecked] = React.useState(false)
-  const [twoChecked, setTwoChecked] = React.useState(false)
+  const [checked, setChecked] = React.useState(false);
+  const [twoChecked, setTwoChecked] = React.useState(false);
+  const [threeChecked, setThreeChecked] = React.useState(false);
+
   return (
     <>
+      <p><b>These are examples of single (not-grouped) Radio buttons:</b></p>
+      <br/>
```

Comment:
Any reason we use `<br/>`s instead of wrapping all these sections in `<p>` tags?

Reply:
I've been using <br/> tags to visually space out the sections. without them everything is crammed together and hard to read. Not sure what hunter's end plan is for styling the docs, but <br/> tags are easy to remove.